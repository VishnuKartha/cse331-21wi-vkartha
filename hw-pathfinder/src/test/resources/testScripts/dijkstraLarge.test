ind a path with Dijkstra's algorithm with a large graph (the large graph was presented in section)

CreateGraph largeGraph

AddNode largeGraph A
AddNode largeGraph B
AddNode largeGraph C
AddNode largeGraph D
AddNode largeGraph E
AddNode largeGraph F
AddNode largeGraph G
AddNode largeGraph H




AddEdge largeGraph A B 2.0
AddEdge largeGraph A C 1.0
AddEdge largeGraph A D 4.0

AddEdge largeGraph B C 5.0
AddEdge largeGraph B E 10.0
AddEdge largeGraph B F 2.0

AddEdge largeGraph C A 9.0
AddEdge largeGraph C E 11.0

AddEdge largeGraph D C 2.0

AddEdge largeGraph E D 7.0
AddEdge largeGraph E G 1.0

AddEdge largeGraph F H 3.0

AddEdge largeGraph G E 3.0
AddEdge largeGraph G F 2.0

AddEdge largeGraph H G 1.0

FindPath largeGraph A E
