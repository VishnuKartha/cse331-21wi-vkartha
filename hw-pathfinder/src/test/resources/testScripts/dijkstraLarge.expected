Unrecognized command: ind

created graph largeGraph

added node A to largeGraph
added node B to largeGraph
added node C to largeGraph
added node D to largeGraph
added node E to largeGraph
added node F to largeGraph
added node G to largeGraph
added node H to largeGraph




added edge 2.000 from A to B in largeGraph
added edge 1.000 from A to C in largeGraph
added edge 4.000 from A to D in largeGraph

added edge 5.000 from B to C in largeGraph
added edge 10.000 from B to E in largeGraph
added edge 2.000 from B to F in largeGraph

added edge 9.000 from C to A in largeGraph
added edge 11.000 from C to E in largeGraph

added edge 2.000 from D to C in largeGraph

added edge 7.000 from E to D in largeGraph
added edge 1.000 from E to G in largeGraph

added edge 3.000 from F to H in largeGraph

added edge 3.000 from G to E in largeGraph
added edge 2.000 from G to F in largeGraph

added edge 1.000 from H to G in largeGraph

path from A to E:
A to B with weight 2.000
B to F with weight 2.000
F to H with weight 3.000
H to G with weight 1.000
G to E with weight 3.000
total cost: 11.000